examples/planning/train.ipynb

```diff
@@ -0,0 +1,319 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Introduction to Planning for Self Driving Vehicles\n",
+    "\n",
+    "In this notebook you are going to train your own ML policy to fully control a SDV. You will train your model using the Lyft Prediction Dataset and L5Kit\n",
+    "\n",
+    "**Before starting, please download the [Lyft L5 Prediction Dataset 2020](https://self-driving.lyft.com/level5/prediction/) and follow [the instructions](https://github.com/lyft/l5kit#download-the-datasets) to correctly organise it.**\n",
+    "\n",
+    "The policy will be deep neural network (DNN) which will be invoked by the SDV to obtain the next command to execute.\n",
+    "\n",
+    "More in details, you will be working with a CNN architecture based on ResNet50.\n",
+    "\n",
+    "![model](../../images/planning/model.svg)\n",
```

Comment:
Sorry to mention this now ... but AV is not in the center vertically as we normally do - could this be a reason of confusion?

Reply:
nah, this is really just illustrative